content/rc/cloud-integrations/aws-marketplace/aws-bedrock.md

```diff
@@ -0,0 +1,107 @@
+---
+Title: Use Redis Cloud with Amazon Bedrock
+LinkTitle: Amazon Bedrock 
+description: Shows how to use your Redis database with Amazon Bedrock to customize foundational models.
+weight: $weight
+alwaysopen: false
+categories: ["RC"]
+aliases: 
+---
+
+[Amazon Bedrock](https://aws.amazon.com/bedrock/) is a service that allows you to securely customize foundational models (FMs) with your own data, and to use these models without having to build complex infrastructure management. With Amazon Bedrock, users can access FMs from a variety of vendors through a single API, streamlining the process of creating generative artificial intelligence (AI).
+
+Amazon Bedrock allows you to choose Redis Cloud as the [vector database](https://redis.com/solutions/use-cases/vector-database/) for your models. Once your database is set up and connected to Amazon Bedrock, it will import text data from an Amazon Simple Storage Service (S3) bucket into Redis Cloud and use it to extract relevant information when prompted.
```

Comment:
```suggestion
Amazon Bedrock allows you to choose Redis Cloud as the [vector database](https://redis.com/solutions/use-cases/vector-database/) for your models. After your database is set up and connected to Amazon Bedrock, it will import text data from an Amazon Simple Storage Service (S3) bucket into Redis Cloud and use it to extract relevant information when prompted.
```